common/changes/@itwin/property-grid-react/Property-grid-ancestor_2022-06-30-21-29.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/property-grid-react",
+      "comment": "Added ancestor navigation to the Property Grid.",
+      "type": "patch"
```

Comment:
needs to be a minor version bump since its changing the core deps by a minor version

Reply:
Done